packages/swingset-liveslots/src/vatDataTypes.d.ts

```diff
@@ -97,15 +96,36 @@ export type DefineKindOptions<C> = {
 
   /**
    * Intended for internal use only.
+   * Only applicable if this is a class kind. A class kit kind should use
+   * `interfaceGuardKit` instead.
+   *
    * If an `interfaceGuard` is provided, then the raw methods passed alongside
    * it are wrapped by a function that first checks that this method's guard
    * pattern is satisfied before calling the raw method.
    *
-   * In `defineDurableKind` and its siblings, this defaults to off.
-   * `prepareExoClass` use this internally to protect their raw class methods
+   * In `defineDurableKind` and its siblings, this defaults to `undefined`.
+   * Exo classes use this internally to protect their raw class methods
    * using the provided interface.
+   * In absence, an exo is protected anyway, while a bare kind is
+   * not (detected by `!thisfulMethods`),
+   */
+  interfaceGuard?: InterfaceGuard;
+
+  /**
+   * Intended for internal use only.
+   * Only applicable if this is a class kit kind. A class kind should use
+   * `interfaceGuard` instead.
+   *
+   * If an `interfaceGuardKit` is provided, then each member of the
+   * interfaceGuardKit is used to guard the corresponding facet of the
+   * class kit.
+   *
+   * In `defineDurableKindMulti` and its siblings, this defaults to `undefined`.
+   * Exo class kits use this internally to protect their facets.
+   * In absence, an exo is protected anyway, while a bare kind is
+   * not (detected by `!thisfulMethods`),
    */
-  interfaceGuard?: InterfaceGuard<unknown>;
+  interfaceGuardKit?: Record<string, InterfaceGuard>;
```

Comment:
good improvement.

this could be typed such that it takes _either_ `interfaceGuard` or `interfaceGuardKit` but I don't think it's worth it for this internal use.

Reply:
incidentally, I find "Kit" as shorthand for "multifaceted" to be loose. I wonder if we can come up with a good word that is different from "kit" as used in "issuerKit" (foo and related).

Reply:
Would love to some suggested alternatives to "Kit". But we use it a lot so please short. In any case, because we use it a lot, it would need to be much better to be worth changing.

Resolving.